JapaneseFilter/sections/specific.txt

```diff
@@ -15,6 +15,7 @@ kyoko-np.net##div[class^="col-"] > div.ad
 sankei.com###sponsored
 ganganonline.com##header#sqexHeader-black_rsp
 asahi.com##.AdMod
+japanese.engadget.com##p.partner_container
```

Comment:
that exclusion in our Eng filter. The rule with comment, so you can check if it is still required.

Reply:
yea, it's still required, but it's not a problem anymore